firebase/firebase.js

```diff
@@ -0,0 +1,16 @@
+import firebase from 'firebase/app';
+import 'firebase/database';
+
+const config = {
```

Comment:
I believe for now this is perfectly fine, but moving forward, having a config outside of a codebase is much better as it does not interfere with codebase business logic. And also, can cater different environments.